### PR TITLE
Feature/gasvm refactoring/cosmetic changes

### PR DESCRIPTION
**Pull request by Regis128**

### CODE REVIEW

File: src/Spectre.libClassifier/Svm.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```
Comment (gmrukwa) [inline src/Spectre.libClassifier/Svm.h]: Sometimes I wonder, what does this git find different in these lines... @mg6? I checked it twice yesterday - there's even no whitespace difference...

File: src/Spectre.libClassifier/UnsupportedDatasetTypeException.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-* UnsupportedDatasetTypeException.h
+* UnsupportedDatasetTypeExceotion.h
```
Comment (gmrukwa) [inline src/Spectre.libClassifier/UnsupportedDatasetTypeException.h]: Typo. Exce**o**tion.

Comment (Regis128) [reply src/Spectre.libClassifier/UnsupportedDatasetTypeException.h]: I will fix it in next PR ok? I know maybe not the best strategy but to fasten the process of merging

Comment (Regis128) [reply src/Spectre.libClassifier/UnsupportedDatasetTypeException.h]: Or I think it is fixed in few later commits which aren't yet in allCommit commit ;] (DownSampledOpenCvDataset)

Comment (gmrukwa) [reply src/Spectre.libClassifier/UnsupportedDatasetTypeException.h]: If you see such errors, do not stage such lines to the commit please.

File: src/Spectre.libClassifier/Svm.h (hunk 1 above)
Comment (mg6) [reply src/Spectre.libClassifier/Svm.h]: Actually, file encoding has changed (BOM removed), so there was a whitespace difference.
```diff
% git show edcdabc -- src/Spectre.libClassifier/Svm.h
commit edcdabcaef5ee3b86ad2e8ddb6c6927d89e957fd
Author: Regis128 <regis1286@gmail.com>
Date:   Thu Jan 4 21:39:21 2018 +0100

    Fixed indents in Svm

diff --git a/src/Spectre.libClassifier/Svm.h b/src/Spectre.libClassifier/Svm.h
index a990b80..2499492 100644
--- a/src/Spectre.libClassifier/Svm.h
+++ b/src/Spectre.libClassifier/Svm.h
@@ -1,4 +1,4 @@
-<U+FEFF>/*
+/*
 * Svm.h
 * Simple wrapper for OpenCV SVM.
 *
(redacted)
```

**Comment by gmrukwa** [comment]
[Merge updates of GA](https://trello.com/c/rHY973Dc/344-merge-updates-of-ga)

**Pull request merged**